e2e/test/D2cMessage.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Microsoft.Azure.Devices.E2ETests
+{
+    public class D2cMessage
```

Comment:
Since it can represent either C2D or D2C messages, rename it to something more generic, `TestMessage` maybe?